sql/telemetry_derived/glam_valid_builds_v1/query.sql

```diff
@@ -0,0 +1,14 @@
+WITH build_id_user_counts AS (
```

Comment:
A docstring comment would be nice: `-- Calculate the top two build ids per day over the last year.`